include/fw_metadata.h

```diff
@@ -184,11 +185,13 @@ static inline bool memeq(const void *expected, const void *actual, u32_t len)
 	}
 }
 
-/* Get a pointer to the firmware_info structure inside the firmware. */
+/* Check and provide a pointer to a firmware_info structure.
+ *
+ * @return pointer if valid, NULL if not.
+ */
 static inline const struct fw_firmware_info *
-fw_firmware_info_get(u32_t firmware_address)
+fw_firmware_info_check(u32_t finfo_addr)
```

Comment:
Just a suggestion for variable name feel free to ignore
```suggestion
fw_firmware_info_check(u32_t fw_info_addr)